x-pack/plugins/security_solution/scripts/endpoint/action_responder/utils.ts

```diff
@@ -144,6 +147,40 @@ export const sendEndpointActionResponse = async (
     }
   }
 
+  // For `get-file`, upload a file to ES
+  if (action.command === 'get-file' && !endpointResponse.error) {
+    // Add the file's metadata
+    const fileMeta = await esClient.index<UploadedFile>({
+      index: FILE_STORAGE_METADATA_INDEX,
+      id: `${action.id}.${action.hosts[0]}`,
+      body: {
+        file: {
+          created: new Date().toISOString(),
+          extension: 'zip',
+          path: '/some/path/bad_file.txt',
+          type: 'file',
+          size: 221,
+          name: 'bad_file.txt.zip',
+          mime_type: 'application/zip',
+          Status: 'READY',
+          ChunkSize: 4194304,
+        },
+      },
+      refresh: 'wait_for',
+    });
+
+    await esClient.index({
+      index: FILE_STORAGE_DATA_INDEX,
+      id: `${fileMeta._id}.0`,
+      body: {
+        bid: fileMeta._id,
+        last: false,
+        data: 'UEsDBBQACAAIAFVeRFUAAAAAAAAAABMAAAAMACAAYmFkX2ZpbGUudHh0VVQNAAdTVjxjU1Y8Y1NWPGN1eAsAAQT1AQAABBQAAAArycgsVgCiRIWkxBSFtMycVC4AUEsHCKkCwMsTAAAAEwAAAFBLAQIUAxQACAAIAFVeRFWpAsDLEwAAABMAAAAMACAAAAAAAAAAAACkgQAAAABiYWRfZmlsZS50eHRVVA0AB1NWPGNTVjxjU1Y8Y3V4CwABBPUBAAAEFAAAAFBLBQYAAAAAAQABAFoAAABtAAAAAAA=',
```

Comment:
for :hankey: and :rofl:  I actually checked what this is.

It's a zip of a single file, `bad_file.txt` with contents: `this is a bad file`.

Reply:
LOL - yep 😎 

(and thanks for testing that out)